draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2663,8 +2688,8 @@ https://video.example.com/k/movie1/4/013.ts
          <figure>
             <artwork><![CDATA[
 {
-  "start": 2000-01-01T02:30:00.00,
-  "end": 2000-01-01T04:30:00.00,
+  "start": "2000-01-01T02:30:00.00",
+  "end": "2000-01-01T04:30:00.00"
 }
          ]]></artwork>
           </figure>
```

Comment:
For DateLocalTime below, Kevin provided the following feedback:

> section 8.2.3: seems like a lot of effort just to get a datetime with no timezone.  can we not just use ISO8601 localtime?  https://en.wikipedia.org/wiki/ISO_8601#Local_time_(unqualified)

Reply:
Will ask ori

Reply:
Hmm, giving this some thought, so the LocalTimeWindow concept makes complete sense to me.  Kevin's comment may make a little less sense to me, although I think what he is saying here is that we're spending this effort defining a time format in 8.2.3 when we could just use ISO 8601.  And, I think the answer to this is that maybe we're trying to make this easier for the dCDN to parse and understand, as ISO 8601 is more permissive when it comes to things like DST, as well as variable precision in how time is formatted.  Do you think my understanding is correct here?

Reply:
I believe you are correct.  Lets open it for brouder discussion